src/containers/Tablets/Tablets.tsx

```diff
@@ -163,19 +170,20 @@ function TabletActions(tablet: TTabletStateInfo) {
 
 interface TabletsProps {
     path?: string;
+    database?: string;
     nodeId?: string | number;
     className?: string;
 }
 
-export function Tablets({nodeId, path, className}: TabletsProps) {
+export function Tablets({nodeId, path, database, className}: TabletsProps) {
     const [autoRefreshInterval] = useAutoRefreshInterval();
 
     let params: TabletsApiRequestParams = {};
     const node = nodeId === undefined ? undefined : String(nodeId);
     if (node !== undefined) {
-        params = {nodes: [String(node)]};
+        params = {nodeId: node, database};
     } else if (path) {
-        params = {path};
+        params = {path, database};
```

Comment:
It looks like `database` param has higher priority, than `nodeId` param. So when we open Node Tablets tab, we receive database tablets, even if they don't belong to current node (database can have many nodes)

I suggest using only `nodeId` param, if it's present, and `database` param otherwise